packages/desktop-gui/src/settings/settings.jsx

```diff
@@ -34,6 +35,9 @@ const Settings = observer(({ project, app }) => {
           <Panel header='Proxy Settings' key='proxy-settings' className='form-horizontal settings-proxy'>
             <ProxySettings app={app} />
           </Panel>
```

Comment:
"Proxy Settings" is technically a "Global Setting", at some point, we should move the preferred editor + proxy settings to their own "Global Settings" section. 

So we don't overly inflate the UI changes from this PR, maybe the "Proxy Settings" should just be moved into the new "Global Settings" section for now?